### DISCUSSION TITLE
Nested mergerfs?

### DISCUSSION THREAD

**Discussion by iamwoz** [trapexit/mergerfs#1034] [General]
Hi, wondering if I can use use mergerfs to combine two existing mergerfs mounts into a single.

I have two separate mergerfs mounts, one based on XFS volumes, the other based on BTRFS volumes. I'm in the process of migrating from XFS to BTRFS. Having a another mergerfs other the top of these two would help me with my applications maintaining access to file during this transition.

Thanks for mergerfs - came from UnRaid and couldn't be happier!

**Comment by trapexit** [comment]
You can but I wouldn't recommend it as a daily driver. What's the problem with just using one mount?

**Reply by iamwoz** [reply]
I might be overcomplicating this. Intend to move to storage tiers with scripted time expiry moves - was using this migration of filesystems to start breaking out into separate tiers but maybe I should do that as another exercise once all my filesystems are sorted

Am also using MergerFS to balance the drives at the same time: rsync from drive 1 (in pool x) to pool y

**Comment by hongkongkiwi** [comment]
I also have this question, I have a pool mount which balances writes across multiple drives (to do a sort of wear levelling on the SSDs), then I want to include the pool mount in another mount so for example:
/disk1:/disk2:/disk3 = pool (balance)
/pool:/cloud = data (ff)

So I want to make sure that anything I write to data first uses my pool and balances across those, otherwise it writes to cloud.

Is there an alternative approach apart from nesting here that could get me the same results? is there any issues with doing this?

**Reply by trapexit** [reply]
> Is there an alternative approach apart from nesting here that could get me the same results?

No. Not currently. Maybe mergerfs v3 will support something like this but I'm still, slowly, working on it.

> is there any issues with doing this?

Define "issues". You are adding a lot more overhead.
